packages/devtools-reps/src/object-inspector/index.js

```diff
@@ -148,21 +167,43 @@ class ObjectInspector extends Component {
   }
 
   state: State;
+
+  shouldComponentUpdate(nextProps : Props, nextState: State) {
```

Comment:
nit: do we have style-guide for @flow syntax? I am seeing that you have a space between the `nextProps` and `: ` but not after `nextState`

Similar inconsistency is when you define a type for function return value.

Reply:
There is no style guides, I try to keep it consistent with the code (and missed this one)
It would be nice if we could enforce this with eslint